dev/breeze/src/airflow_breeze/commands/configuration_and_maintenance_commands.py

```diff
@@ -157,9 +157,9 @@ def cleanup(verbose: bool, dry_run: bool, github_repository: str, all: bool, ans
         )
         images = command_result.stdout.splitlines() if command_result and command_result.stdout else []
         if images:
-            get_console().print("[light_blue]Removing images:[/]")
+            get_console().print("[info]Removing images:[/]")
```

Comment:
Should these use the new Enum too?

Reply:
Hmm. I think in most cases "[info]...." is a bit better than f"{INFO.value}....." + import :).

The only place where it makes sense to use enum is where it is programmatically passed (as it is in ci_group as parameter, then it's worth to use enum  :)